recipes/ace_cream/build.sh

```diff
@@ -0,0 +1,6 @@
+if [ $OSX_ARCH ]; then
+export LDFLAGS=${LDFLAGS}" "-undefined" "dynamic_lookup
+else
+export LDFLAGS=${LDFLAGS}" "-shared
+fi
+$PYTHON setup.py install
```

Comment:
You should use `pip` as shown in:

https://conda-forge.org/docs/meta.html#use-pip

but adapted to the `build.sh` script. Same goes for `bld.bat`